### ISSUE TITLE
Incorrect function of the return key in W10

### ISSUE THREAD

**Issue by New-Pilgrim** [wxMaxima-developers/wxmaxima#1148]
Incorrect functioning of the return key in W10.  Both return and shift return are forceing calculation of the cell and the cursor then moves to open a new cell.  Changing the configuration setting makes no difference.  It is at the present impossible to insert new lines within a cell.  This is in explicit contradiction of the guide notes in the hint file.
Note, the same problem does not arise when running wxmaxima under Ubuntu 18.04.
Setup details:
wxMaxima version: 19.05.7
using wxWidgets version: wxWidgets 3.1.0
Maxima version: 5.43.0
Maxima build date: 2019-06-01 18:55:31
Host type: x86_64-w64-mingw32
System type: Win32 10.0.18362 X86-64
Lisp implementation type: SBCL
Lisp implementation version: 1.4.14
wxMaxima's idea of the directory layout is:
User configuration dir: C:/Users/MS2/maxima/
Help dir: D:\maxima-5.43.0\wxMaxima/help

**Comment by gunterkoenigsmann** [comment]
Does Ctrl+Enter work for you - and what is the config-dialogue setting for "Enter evaluates cells"?

**Comment by New-Pilgrim** [comment]
No.  Ctrl+Enter and Enter produce the same result, i.e. evaluation + new cell and not new line.  Changing the config-dialogue setting for "Enter evaluates cells" makes no difference.  
The only means of entering multiline cells is by cutting and pasteing from a notepad!

**Comment by gunterkoenigsmann** [comment]
In the meantime I got hold of a Win10 computer - where wxMaxima worked fine. Which means there is a difference between our computers. I dislike bugs I cannot reproduce.

**Comment by New-Pilgrim** [comment]
Gunter,

Thank you for your attention. I share your distaste.

In response to your e-mail I loaded wxMaxima on to another W10 PC.  And 
like you, I do not get the same problem as that that I identified.

On the W10 PC, on which the problem was identified, I have reloaded 
wxMaxima, but the fault persists.  Now whether, in some way, the fault 
is specific to that PC, or is the result of the subtly different 
flavours of W10 resulting from Microsoft's extended roll-outs of the W10 
version updates, or other parts of the software environment, I cannot tell.

Regards

Adrian Howard-Jones
